### DISCUSSION TITLE
"-thumb" required for thumbnails?

### DISCUSSION THREAD

**Discussion by haydenbroadhead** [jellyfin/jellyfin#12079] [General] [answered]
Since when do videos need to have a name "-thumb" in the suffix? 

For context I've got a yt-dlp script downloading thumbnails with the exact name but obviously ".jpg". And last night my weekly docker update for jellyfin has made all downloaded videos after this without thumbnails. I've identified what the cause is now... no big deal, but just curious when it changed? I can't find a reference to it.

Maybe this was added a while ago but stopped working for same name thumbnails recently.

[Link to docs](https://jellyfin.org/docs/general/server/media/shows#thumb)

**Comment by Shadowghost** [comment, marked as answer]
There were some changes in the last update to fix an inconsistency with exported vs imported image naming but the fix introduced another bug. Should be fixed again in the next version. Sorry for the inconvenience.

**Reply by haydenbroadhead** [reply]
Ah that makes sense, thanks!
